src/borg/xattr.py

```diff
@@ -68,7 +68,7 @@ def get_all(path, follow_symlinks=True):
         libc_name = 'libc.dylib'
     else:
         msg = "Can't find C library. No fallback known. Try installing ldconfig, gcc/cc or objdump."
-        logger.error(msg)
+        print(msg, file=sys.stderr)  # logger isn't initialized at this stage
```

Comment:
oh, oops.

so you found that in a traceback because it could find your C library and then it crashed when trying to log that?